src/langchain_google_cloud_sql_pg/engine.py

```diff
@@ -747,6 +750,91 @@ def init_document_table(
             )
         )
 
+    async def _ainit_checkpoint_table(
+        self,
+        schema_name: str = "public",
+        checkpoints_table_name: str = CHECKPOINTS_TABLE,
+        checkpoint_writes_table_name: str = CHECKPOINT_WRITES_TABLE,
```

Comment:
Can we simplify this to 
```suggestion
        table_name: str = CHECKPOINTS_TABLE,
        writes_table_name: str = CHECKPOINT_WRITES_TABLE,
```